packages/management/tkr-conversion-webhook/metadata.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: data.packaging.carvel.dev/v1alpha1
+kind: PackageMetadata
+metadata:
+  name: tkr-conversion-webhook.tanzu.vmware.com
+  namespace: tkr-conversion-webhook
+spec:
+  displayName: "tkr-conversion-webhook"
+  longDescription: "Conversion Webhook for which allows for lossless conversion between v1alpha1 and v1alpha3 versions of TKR API."
```

Comment:
Question: Would it be lossless conversion? If user tries to access a TKR in v1alpha3 suing v1alpha1 api version API? and vice-versa? (eg: pause image)

Reply:
Hmm, I think this depends on what the definition of lossless is. 
- `v1alpha1` -> `v1alpha3` will definitely have a loss due to incompatibility of the formats like in the example you've pointed out: `v1alpha1` can have multiple pause images in a list, while `v1alpha3` has the scope for one and only one pause image.
- However, the round-trip conversion is lossless because of the use of annotations. Taking the same example, `v1alpha1` -> `v1alpha3` -> `v1alpha1` will have all the pause images intact.

Reply:
Fixed in e5129ee68af5451552a1d7e08001f02c6160fc1e